### PR TITLE
Add info about sw subscription on deployment

### PR DESCRIPTION
**Pull request by tznamena**
This PR adds a notification alert to the UI to notify user that they need a sw subscription. The alert can be dismissed by clicking on a link at the bottom of the alert.
Also updated NodeJS version.

## Screenshot

Screenshot from 2023-07-13 12-47-06

### CODE REVIEW

**Comment by prigit1** [comment]
recommend rewording as AAP appears 5 times in the text : 

Ansible Automation Platform Deployment Engine
---------------------------------
Ansible Automation Platform(AAP) requirements
To use AAP on Azure, you must have a valid subscription in your Red Hat account.
Please click here to setup your AAP subscription as well as your Red Hat, account if needed.
Click Dismiss, if you already have it all setup.

File: ui/src/app/Deployment/DeploymentSteps/Steps.tsx [hunk 1]
```diff
@@ -1,23 +1,33 @@
...
+import { DeploymentInfo } from '../DeploymentInfo';
 
 interface IDeploymentStepsProps {
   stepsData: DeploymentStepData[]
 }
 
 
 export const DeploymentSteps = ({ stepsData }: IDeploymentStepsProps, ) => {
+
+  // TODO: Persist this value somewhere in current user's session
```
[the remark this answers is not in the document]
Comment (tznamena) [reply ui/src/app/Deployment/DeploymentSteps/Steps.tsx:16]: Yes, a cookie, local storage (in browser) or even API call to store it server side (DB). I didn't want to add it to this PR to keep it small and get reviews for the user interface changes first.

File: ui/src/app/Deployment/DeploymentInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      //actionClose={<AlertActionCloseButton onClose={} />}
+      actionLinks={
+        <React.Fragment>
+          {/* <AlertActionLink onClick={() => alert('Clicked on View details')}>Go to console.redhat.com</AlertActionLink> */}
+          <AlertActionLink onClick={closeHandler}>Dismiss, I already have it all setup</AlertActionLink>
+        </React.Fragment>
+      }
+    >
+      <p>To use Ansible Automation Platform on Azure, you must have a valid subscription for Ansible Automation Platform in your Red Hat account.</p>
+      <p>You can click on <a href='https://console.redhat.com' target='_blank' rel="noreferrer">this link</a> to setup your Ansible Automation Platform subscription as well as your Red Hat, account if needed.</p>
```
Comment (ariordan-redhat) [inline ui/src/app/Deployment/DeploymentInfo.tsx:25]:
```suggestion
      <p>You can set up your Ansible Automation Platform subscription and your Red Hat account on the </p> <a href='https://console.redhat.com' target='_blank' rel="noreferrer">Red Hat Hybrid Cloud Console</a>.
```

File: ui/src/app/Deployment/DeploymentSteps/Steps.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (tznamena) [reply ui/src/app/Deployment/DeploymentSteps/Steps.tsx]: Using browser's session storage for now. It lives as long as tab is open, survives page reloads and each tab has its own session(even if loading same page).

**Review by bhavenst — APPROVED** [review]
Looks great, works great.

**Pull request merged**